.evergreen/serverless/create-instance.sh

```diff
@@ -9,6 +9,7 @@ set +o xtrace # Disable xtrace to ensure credentials aren't leaked
 #   SERVERLESS_DRIVERS_GROUP    Required. Atlas group for drivers testing.
 #   SERVERLESS_API_PUBLIC_KEY   Required. Public key for Atlas API request.
 #   SERVERLESS_API_PRIVATE_KEY  Required. Private key for Atlas API request.
+#   SERVERLESS_SKIP_CRYPT       Optional. If set, skips installing mongocryptd and crypt_shared (defaults to true)
```

Comment:
do you assign this anywhere?

Reply:
The variable is only used in line 118. It is not used on Serverless and only there for people that want to run `create-instance.sh` locally where they already have mongodb and everything installed and want to skip downloading it again.